corehq/apps/reports/standard/deployments.py

```diff
@@ -507,8 +507,8 @@ def _fmt_timestamp(timestamp):
         result[0][1] = table
         return result
 
-    def get_location_column(self, user_doc, user_loc_dict):
-        user = CouchUser.wrap_correctly(user_doc)
+    def _get_location_column(self, user_doc_es, user_loc_dict):
```

Comment:
nit: user_es_doc?

Reply:
Addressed in [e5c2342](https://github.com/dimagi/commcare-hq/pull/34744/commits/e5c23424c947b4a6e128d6eeca5c9d628c884ffc). I also renamed `user_doc` to `user_es_doc` in the loop to make it more explicit.